comp/virtual-comp/code-submitter.yaml

```diff
@@ -17,4 +17,3 @@ description: >-
   Required actions:
    - [ ] Setup the code submitter on the SR's infrastructure
    - [ ] Populate the code submitter with the teams' passwords
-   - [ ] Advertise the code submitter to the teams
```

Comment:
We still need to do this. I can't find where it's moved to.

Reply:
:+1: good catch, I think this should be in the announcement task.

79965f7 (deliberately not a separate checkbox though as it's not really a separate thing to advertise)